protobuf/syft_proto/execution/v1/type_wrapper.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package syft_proto.execution.v1;
+option java_package = "org.openmined.syftproto.execution.v1";
+
+message type {
```

Comment:
We might want to give this a more descriptive name. I'm not entirely sure what that should be though.

Reply:
Hmm, what do you think about:
* native_type
* class_type <- my favorite
* basic_type

What do you think about those?

Reply:
`class_type` seems good. 👍